examples/with_winit/src/lib.rs

```diff
@@ -136,397 +136,412 @@ fn run(
-                                        * transform;
+    event_loop
+        .run(move |event, event_loop| match event {
+            Event::WindowEvent {
+                ref event,
+                window_id,
+            } => {
+                let Some(render_state) = &mut render_state else {
+                    return;
+                };
+                if render_state.window.id() != window_id {
+                    return;
+                }
+                match event {
+                    WindowEvent::CloseRequested => event_loop.exit(),
+                    WindowEvent::ModifiersChanged(m) => modifiers = m.state(),
+                    WindowEvent::KeyboardInput { event, .. } => {
+                        if event.state == ElementState::Pressed {
+                            match event.logical_key.as_ref() {
+                                Key::Named(NamedKey::ArrowLeft) => {
+                                    scene_ix = scene_ix.saturating_sub(1)
                                 }
-                            }
-                            Some(VirtualKeyCode::Space) => {
-                                transform = Affine::IDENTITY;
-                            }
-                            Some(VirtualKeyCode::S) => {
-                                stats_shown = !stats_shown;
-                            }
-                            Some(VirtualKeyCode::D) => {
-                                complexity_shown = !complexity_shown;
-                            }
-                            Some(VirtualKeyCode::C) => {
-                                stats.clear_min_and_max();
-                            }
-                            Some(VirtualKeyCode::M) => {
-                                aa_config_ix = if modifiers.shift() {
-                                    aa_config_ix.saturating_sub(1)
-                                } else {
-                                    aa_config_ix.saturating_add(1)
-                                };
-                            }
-                            Some(VirtualKeyCode::P) => {
-                                if let Some(renderer) = &renderers[render_state.surface.dev_id] {
-                                    if let Some(profile_result) = &renderer
-                                        .profile_result
-                                        .as_ref()
-                                        .or(profile_stored.as_ref())
+                                Key::Named(NamedKey::ArrowRight) => {
+                                    scene_ix = scene_ix.saturating_add(1)
+                                }
+                                Key::Named(NamedKey::ArrowUp) => complexity += 1,
+                                Key::Named(NamedKey::ArrowDown) => {
+                                    complexity = complexity.saturating_sub(1)
+                                }
+                                Key::Character(key @ "a") | Key::Character(key @ "e") => {
```

Comment:
Previously, rotate anti-clockwise was `q`. It might be worth checking whether your coding font makes the difference between Q and A sufficiently clear

Does rotation still work when caps lock is enabled with this change?

Reply:
whoops, thats a typo. fixed.